packages/tables/backend/sirius-components-tables/src/main/java/org/eclipse/sirius/components/tables/components/ColumnComponent.java

```diff
@@ -50,27 +52,44 @@ public Element render() {
     }
 
     private Element doRender(VariableManager variableManager, Object object) {
-        ColumnDescription columnDescription = this.props.getColumnDescription();
+        ColumnDescription columnDescription = this.props.columnDescription();
 
         VariableManager columnVariableManager = variableManager.createChild();
         columnVariableManager.put(VariableManager.SELF, object);
         String targetObjectId = columnDescription.getTargetObjectIdProvider().apply(columnVariableManager);
         String targetObjectKind = columnDescription.getTargetObjectKindProvider().apply(columnVariableManager);
         String label = columnDescription.getLabelProvider().apply(columnVariableManager);
+        Integer initialWidth = columnDescription.getInitialWidthProvider().apply(columnVariableManager);
+        boolean resizable = columnDescription.getIsResizablePredicate().test(columnVariableManager);
         UUID columnId = this.computeColumnId(targetObjectId);
-        this.props.getCache().putColumnObject(columnId, object);
+        this.props.cache().putColumnObject(columnId, object);
 
-        ColumnElementProps columnElementProps = ColumnElementProps.newColumnElementProps(columnId)
+        ColumnElementProps.Builder columnElementProps = ColumnElementProps.newColumnElementProps(columnId)
```

Comment:
You can just compute the value of `width` before hand and why can't width be initialized to initialWidth?